test-cases/performance/perf-regression-latency-250gb-with-nemesis.yaml

```diff
@@ -15,7 +15,7 @@ n_monitor_nodes: 1
 
 instance_type_loader: 'c5.2xlarge'
 instance_type_monitor: 't3.large'
-instance_type_db: 'i3.2xlarge'
+instance_type_db: 'i3en.2xlarge'
```

Comment:
That's about 50% more expensive - do we need it?

Reply:
That's part of the requirements since the CPU in i3en is much better.
I believe that soon we will need to move to i4i instance anyway.